backend/attendance-service/src/main/java/org/egov/web/models/StaffPermissionResponse.java

```diff
@@ -21,7 +23,7 @@ public class StaffPermissionResponse {
     private ResponseInfo responseInfo = null;
 
     @JsonProperty("staff")
-    private StaffPermission staff = null;
+    private List<StaffPermission> staffPermissionResponseList = null;
```

Comment:
Let the variable name be just `staff`.

Reply:
staffPermissionResponseList renamed to staff